Swipeable/SwipeableFlatList.js

```diff
@@ -27,6 +27,7 @@ import SwipeableRow from './SwipeableRow';
 //      * Callback method to render the view that will be unveiled on swipe
 //      */
 //     renderQuickActions: renderItemType,
+//     disableSwipe?: ?boolean,
```

Comment:
Could you also add a line in the README.md about this new prop so it's not undocumented. 🙏

Reply:
DONE